dist.sh

```diff
@@ -8,6 +8,7 @@ allplatforms="windows linux osx"
 platforms="all"
 bname="$(basename $0)"
 tag="$(git describe --tags)"
+version="$(sed 's/-.*//' <<< $tag)"
```

Comment:
I feel like you should just use `$tag` instead. If it's a released version, there will be no commit suffix. And if it's not, the commit suffix should be left there to clarify that it's a development release.